targets/targets.json

```diff
@@ -7327,12 +7335,12 @@
             "FLASH"
         ],
         "public": false,
-        "supported_toolchains": ["GCC_ARM", "ARM", "IAR"],
+        "supported_toolchains": ["GCC_ARM", "ARM"],
         "post_binary_hook": {
             "function": "RTL8195ACode.binary_hook",
             "toolchains": ["ARM_STD", "GCC_ARM", "IAR"]
         },
-        "release_versions": ["5"],
+        "release_versions": [],
```

Comment:
this should still have version 5 ?

Reply:
If it does, I think it should come in via a followup PR, considering how much this PR is blocking.

Reply:
just final check :) LGTM